crates/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -83,8 +74,9 @@ pub fn insert_packages(
 /// Remove package names from the `[install]` table of a manifest
 pub fn remove_packages(
     manifest_contents: &str,
-    pkgs: impl Iterator<Item = String>,
+    pkgs: &[String],
 ) -> Result<Document, TomlEditError> {
+    println!("MANIFEST CONTENTS BEFORE: {}", manifest_contents);
```

Comment:
looks a lot like print debugging :)
if this was intentional to be left in lets at least hide it under debug

```suggestion
    debug!("MANIFEST CONTENTS BEFORE: {}", manifest_contents);
```

Reply:
Also accidentally left in 😅

Reply:
Also accidentally left in 😅